### DISCUSSION TITLE
Meme Sources

### DISCUSSION THREAD

**Discussion by Dragon-0609** [ani-memes/AMII#140] [General]
[link
![unknown](https://amii-assets.s3.amazonaws.com/visuals/alert/pointing_two.gif)
](https://amii-assets.unthrottled.io/assets/view/b2d24e9481c5ff7a116457b03fc03a28)

The anime is `Hanamaru Kindergarten`, she is `Hiiragi`

![photo](https://fastly.4sqi.net/img/general/600x600/c3NRT5UplLhDYiKjM7fqc_AjFKCwcwhxd8mbbkl0mDI.jpg)

-----------

[link
![unknown](https://amii-assets.s3.amazonaws.com/visuals/happy/confetti_celebration.gif)
](https://amii-assets.unthrottled.io/assets/view/a89d09ecb23b324d9d894565d1bd8d7d)

The manga is `Menhera Shoujo Kurumi-chan`, she is `Menhera-chan`

![photo](https://chpic.su/_data/stickers/l/LINE_Menhera_chan_ENG/LINE_Menhera_chan_ENG_047.webp)

-----------

[link
![unknown](https://amii-assets.s3.amazonaws.com/visuals/enraged/enraged_gif_one.gif)
](https://amii-assets.unthrottled.io/assets/view/643d73e0e7e382b9e3439a9b57e3e1e2)

The anime is `Wataten!: An Angel Flew Down to Me`, she is `Hana Shirosaki`

![photo](https://img.cdn.nimg.jp/s/nicovideo/thumbnails/36971722/36971722.79183922.original/r1280x720l?key=44da258f5ca14b1814d1d26ef2bf129de064c849777bac9064107d6339549b35)

**Comment by Unthrottled** [comment]
Updated, thanks again for taking the time to look these up for me!

**Reply by Dragon-0609** [reply]
Glad to help

**Comment by Dragon-0609** [comment]
[link
![unknown](https://amii-assets.s3.amazonaws.com/visuals/waving/wavingReactionOne.gif)
](https://amii-assets.unthrottled.io/assets/view/5d3cdb79cecb701cf5b54a34dc47d63a)

The anime is `Akashic Records of Bastard Magic Instructor`, she is `Sistine Fibel`

![photo](https://sun6-22.userapi.com/impf/c636325/v636325953/5cc86/3SdJc1mLuQo.jpg?size=427x604&quality=96&sign=806d35c393f8c8d185e781b1f290bc6f&type=album)

**Reply by Unthrottled** [reply]
Updated, thank you!

Hey, if you have some assets you'd like to add. You can post them here and when I get the chance (I don't need sound, they can just be animated gifs), I can add them so they'll show up in the plugin sometime. (Provided they are appropriate :)

**Reply by Dragon-0609** [reply]
OK. When I find free time, I'll look for memes.

**Comment by Dragon-0609** [comment]
I have found some memes. All of them are from `Senryu girl`. The girl is `Nanako Yukishiro`.

Panic:
[link to the source where you can get smaller version or even mp4 version
![panic](https://c.tenor.com/axBUKGpKjq0AAAAd/senryu-girl-senryu-shoujo.gif)
](https://tenor.com/view/senryu-girl-senryu-shoujo-nanako-anime-cute-shocked-gif-14302782)

---

Sad:
[link to the source where you can get smaller version or even mp4 version
![sad](https://c.tenor.com/wqoUKp66t0UAAAAd/nanako-yukishiro-bunny.gif)
](https://tenor.com/view/nanako-yukishiro-bunny-sad-sighs-gif-14282091)

---

Congratulations:
[link to the source where you can get smaller version or even mp4 version
![congratulations](https://c.tenor.com/xj0PsiYsUj0AAAAC/senryu-shoujo-senryu-girl.gif)
](https://tenor.com/view/senryu-shoujo-senryu-girl-bravo-clapping-hands-clap-gif-17368043)

---

Angry
[link to the source where you can get smaller version or even mp4 version
![angry](https://c.tenor.com/16Xx09MNDVQAAAAC/senryuu-shojo-angry.gif)
](https://tenor.com/view/senryuu-shojo-angry-anime-girl-angry-eyes-gif-14048233)

---

He is `Eiji Busujima`. Smile:
[link to the source where you can get smaller version or even mp4 version
![smile](https://c.tenor.com/Ov0Dd7j9SPsAAAAd/senryuu-shoujo-senryuu.gif)
](https://tenor.com/view/senryuu-shoujo-senryuu-eiji-busujima-nanako-gif-14989784)

---
---

Starting from here, the anime of the memes is `Saekano: How to Raise a Boring Girlfriend`.

The blonde girl is `Eriri Spencer Sawamura`, another girl is `Utaha Kasumigaoka`, he is `Tomoya Aki`. Furious:
[link to the source where you can get smaller version or even mp4 version
![furious](https://c.tenor.com/WhgtAYw6SrgAAAAd/saekano-aki-tomoya.gif)
](https://tenor.com/view/saekano-aki-tomoya-eriri-spencer-sawamura-kasumikaoka-utaha-saenai-heroine-no-sodatekata-gif-18688632)

---

She is `Megumi Kato`. Idle:
[link to the source where you can get smaller version or even mp4 version
![Idle](https://c.tenor.com/Rp0--QI6fhUAAAAC/katou-megumi-cute.gif)
](https://tenor.com/view/katou-megumi-cute-anime-kawaii-playing-gif-12906735)

**Reply by Unthrottled** [reply]
> He is Eiji Busujima. Smile:

Which emotion is this supposed to portray? Is it a positive or negative emotion? I guess what I am trying to ask, is when is the appropriate time to use this gif?

**Reply by Unthrottled** [reply]
> Starting from here, the anime of the memes is Saekano: How to Raise a Boring Girlfriend.

I know this one! I enjoyed it, Utaha-senpai is best girl.

**Reply by Unthrottled** [reply]
Also before I check, do any of the links you've provided have sound (do I need to check)?

**Reply by Unthrottled** [reply]
I also stumbled across a bunch more gifs I wanted to add. I'll be adding stuff as the week goes on. Thanks again for all the help!

**Reply by Dragon-0609** [reply]
> Which emotion is this supposed to portray? Is it a positive or negative emotion? I guess what I am trying to ask, is when is the appropriate time to use this gif?

I'm not sure about it. If you show it as negative emotion, it'll seem like sarcasm. But if you show it as positive emotion, it'll be strange. I think it'll be better to show it as negative (as sarcasm).

**Reply by Dragon-0609** [reply]
> Also before I check, do any of the links you've provided have sound (do I need to check)?

I have checked some links. They don't have sounds.

**Reply by Unthrottled** [reply]
Fairly sure that I've got all this uploaded now, thank you!

**Comment by Dragon-0609** [comment]
The anime of memes below: `Wotakoi: Love is Hard for Otaku`. She is `Narumi Momose`.

Everything is OK:
[link to the source where you can get smaller version or even mp4 version
![OK](https://c.tenor.com/gzJVthMvc4EAAAAC/wotaku-ni-koi-wa-muzukashii-wotakoi.gif)
](https://tenor.com/view/wotaku-ni-koi-wa-muzukashii-wotakoi-gutonu-guto-nunes-gif-22205971)

---

Sad:
[link to the source where you can get smaller version or even mp4 version
![Sad](https://c.tenor.com/_CZBXYRPoZoAAAAC/otaku-otaku-otaku.gif)
](https://tenor.com/view/otaku-otaku-otaku-narumi-momose-wotaku-ni-koi-wa-muzukashii-not-true-gif-16049365)

---

Idle:
[link to the source where you can get smaller version or even mp4 version
![Idle](https://c.tenor.com/y-8DcOdv9qgAAAAC/narumi-wotaku.gif)
](https://tenor.com/view/narumi-wotaku-happy-driving-gif-13582455)

---

Furious:
[link to the source where you can get smaller version or even mp4 version
![Furious](https://c.tenor.com/GKMszf4mDhEAAAAC/narumi-wotaku.gif)
](https://tenor.com/view/narumi-wotaku-angry-gif-13582451)

**Reply by Unthrottled** [reply]
I've uploaded all of these, and some more.

I've also added this anime to my watch list, it looks really good.

**Reply by Dragon-0609** [reply]
> I've also added this anime to my watch list, it looks really good.

Yep, I have watched it, and I like it.

**Comment by Dragon-0609** [comment]
The anime is `TONIKAWA: Over The Moon For You`. She is `Tsukasa Yuzaki`.

Happy:
[link to the source where you can get smaller version or even mp4 version
![happy](https://c.tenor.com/IgiptP3YYscAAAAC/tonikaku-kawaii-tonikaku.gif)
](https://tenor.com/view/tonikaku-kawaii-tonikaku-tsukasa-anime-anime-happy-gif-22929974)

---

Openning:
[link to the source where you can get smaller version or even mp4 version
![openning](https://c.tenor.com/Yno2YZe2r7kAAAAC/tonikaku-kawaii-dance.gif)
](https://tenor.com/view/tonikaku-kawaii-dance-love-tonikaku-kawaii-gif-18842828)

---

---

The anime is `Horimiya`.  She is `Kyouko Hori`. He is `Izumi Miyamura`.
Agressive:
[link to the source where you can get smaller version or even mp4 version
![agressive](https://c.tenor.com/9eSiGIBP4jMAAAAC/horimiya.gif)
](https://tenor.com/view/horimiya-gif-21679105)

---

Stupid!
[link to the source where you can get smaller version or even mp4 version
![you are stupid](https://c.tenor.com/IyItY8Aoz5kAAAAC/hey-horimiya.gif)
](https://tenor.com/view/hey-horimiya-hori-san-and-miyamura-kun-kyouko-hori-izumi-miyamura-gif-21328499)

---
She is `Remi Ayasaki`. Sorry:
[link to the source where you can get smaller version or even mp4 version
![sorry](https://c.tenor.com/ia4-zMsLLY0AAAAd/anime-girl.gif)
](https://tenor.com/view/anime-girl-crying-pink-haired-gif-23250657)

---

Furious:
[link to the source where you can get smaller version or even mp4 version
![furious](https://c.tenor.com/jnGqbHerOIYAAAAC/hori-is-angry-horimiya.gif)
](https://tenor.com/view/hori-is-angry-horimiya-hori-san-and-miyamura-kun-kyouko-hori-izumi-miyamura-gif-21341936)

---

Greeting (actually it's bye):
[link to the source where you can get smaller version or even mp4 version
![greeting](https://c.tenor.com/0-r9OR4DYsEAAAAC/remi-bye-bye.gif)
](https://tenor.com/view/remi-bye-bye-gif-21600759)

---

What?!
[link to the source where you can get smaller version or even mp4 version
![what](https://c.tenor.com/9KASX7kH8noAAAAC/what-horimiya.gif)
](https://tenor.com/view/what-horimiya-hori-san-and-miyamura-kun-kyouko-hori-izumi-miyamura-gif-21328457)

---

Depression:
[link to the source where you can get smaller version or even mp4 version
![depression](https://c.tenor.com/heASsn9HzUEAAAAC/what-horimiya.gif)
](https://tenor.com/view/what-horimiya-kyouko-hori-izumi-miyamura-slice-of-life-gif-21311025)

**Reply by Unthrottled** [reply]
Great! I've added them all. I've also got `Horimiya` on my watch list as well. Heard good things about it.

Anyways, I don't know how many active users I have at the moment. So pause adding new assets, I'll let the plugin pick up the new assets over the next few working days. I think I'm still below the AWS cloud front free-tier usage.

Thanks again for all the help!

**Comment by Dragon-0609** [comment]
I have found neat thing. When I convert from mp4 to gif, the size is greatly decreased. Can you see difference?
| **Original**  |  **Converted** |
|---|---|
| **2.73 MB**  |  **855 KB** |
|  ![kirito-sao](https://user-images.githubusercontent.com/80271099/166162021-57222de5-7e47-441c-bc61-8fbe68992dcc.gif) | ![kirito](https://user-images.githubusercontent.com/80271099/166162057-f576d4aa-7ab3-4920-9a97-af3e4c104763.gif)  |

Next time do I need to decrease the size as above?

**Reply by Unthrottled** [reply]
> Can you see difference?

Nope, they look the same.

> Next time do I need to decrease the size as above?

Nah, I think you are good. I just visited the tenor.com source you provided and downloaded the gif version there. If they where big (width/height) I would just scale them down.

**Comment by Dragon-0609** [comment]
I have found the source of the meme:

The anime is `Wakaba Girl`. She is `Wakaba Kohashi`.

[link
![unknown](https://amii-assets.s3.amazonaws.com/visuals/celebration/fist_pump_one.gif)
](https://amii-assets.unthrottled.io/assets/view/cc5fb69dc721f92ec409be74f7964a4b)

![picture](https://alchetron.com/cdn/wakaba-girl-3b142986-e7ff-448c-b05b-0e11f5f857d-resize-750.jpeg)

**Reply by Unthrottled** [reply]
Updated, thank you!

**Comment by Dragon-0609** [comment]
The anime is `Banished From The Hero's Party`. He's `Red`, she's `Rit`. Happy:

[link to tenor
![happy](https://c.tenor.com/xlW3RF3bzo8AAAAC/banished-from-the-heros-party-red.gif)
](https://tenor.com/view/banished-from-the-heros-party-red-anime-red-anime-lift-red-and-rit-gif-23512792)

---
---

The anime is `Spy x Family`. She is `Anya Forger`. Sad:

[link to tenor
![sad](https://c.tenor.com/yb2NV8G8gbgAAAAC/spy-x-family-anya.gif)
](https://tenor.com/view/spy-x-family-anya-cry-anime-cry-gif-25175074)

---
He's `Loid Forger`. Idle:
[link to tenor
![idle](https://c.tenor.com/lCJu2KLZQYAAAAAC/anya-spy-x-family.gif)
](https://tenor.com/view/anya-spy-x-family-anya-spy-x-family-pat-gif-25343989)

---
Idle:
[link to tenor
![idle](https://c.tenor.com/2vkg2P2nWVkAAAAC/lonely-anya-spy-x-family.gif)
](https://tenor.com/view/lonely-anya-spy-x-family-anya-taylor-joy-anya-forger-gif-25418081)

---
Boring:
[link to tenor
![boring](https://c.tenor.com/9n08OE8c-b0AAAAd/spy-x-family-spy-family.gif)
](https://tenor.com/view/spy-x-family-spy-family-%E3%82%B9%E3%83%91%E3%82%A4%E3%83%95%E3%82%A1%E3%83%9F%E3%83%AA%E3%83%BC-anya-forger-forger-anya-gif-25353157)

---
Excited:
[link to tenor
![excited](https://c.tenor.com/kmHEH_VM-y4AAAAC/spy-x-family-spy-family.gif)
](https://tenor.com/view/spy-x-family-spy-family-anya-forger-anya-spy-family-bond-forger-gif-25331854)

---
Shocked:
[link to tenor
![shocked](https://c.tenor.com/WJemWYmQXZsAAAAC/spy-x-family-spy-family.gif)
](https://tenor.com/view/spy-x-family-spy-family-spy-family-loid-twilight-%E3%82%B9%E3%83%91%E3%82%A4%E3%83%95%E3%82%A1%E3%83%9F%E3%83%AA%E3%83%BC-gif-25406969)

---
He's `Henry Henderson`. Elegance:
[link to tenor
![elegance](https://c.tenor.com/9PWLGDM1WsAAAAAd/elegance-spy-x-family.gif)
](https://tenor.com/view/elegance-spy-x-family-gif-25573883)

---
Excited:
[link to tenor
![](https://c.tenor.com/G13lUO8AyLIAAAAC/spy-x-family-spy-family.gif)
](https://tenor.com/view/spy-x-family-spy-family-%E3%82%B9%E3%83%91%E3%82%A4%E3%83%95%E3%82%A1%E3%83%9F%E3%83%AA%E3%83%BC-%E3%82%A2%E3%83%BC%E3%83%8B%E3%83%A3-anya-forger-gif-25353159)

---
She's `Yor Forger`. Waiting:
[link to tenor
![waiting](https://c.tenor.com/oWRzl58x2YkAAAAC/spy-x-family-spy-family.gif)
](https://tenor.com/view/spy-x-family-spy-family-yor-forger-yor-briar-yoru-forger-gif-25486803)

---
---
The anime is `Danganronpa`. She's `Junko Enoshima`. Furious:
[link to tenor
![furious](https://c.tenor.com/VpPmDJW4E4wAAAAC/junko-enoshima-danganronpa.gif)
](https://tenor.com/view/junko-enoshima-danganronpa-gif-23013476)

---
The character is `Monokuma`. Party:
[link to tenor
![party](https://c.tenor.com/o3uYO-R8EgAAAAAd/monokuma-dance.gif)
](https://tenor.com/view/monokuma-dance-danganronpa-gif-12895046)

---
She's `Chiaki Nanami`. Idle:
[link to tenor
![idle](https://c.tenor.com/X2KZaNBZLecAAAAC/chiaki-nanami-danganronpa.gif)
](https://tenor.com/view/chiaki-nanami-danganronpa-playing-games-anime-gif-16464598)


I think that's enough.

**Reply by Unthrottled** [reply]
Thanks, I'll have some time this weekend to add these.

**Reply by Unthrottled** [reply]
I've got these uploaded now. I'll eventually get around to watching Spy x Family.

**Comment by Dragon-0609** [comment]
I have cut opening auido of `TONIKAWA: Over The Moon For You` to fit the gif:

[Google Drive link](https://drive.google.com/file/d/1F5fsqV9lz66n4IDQo6rjysHMm8YKDGAz/view?usp=sharing)

**Reply by Unthrottled** [reply]
If you could provide me a link to a video of it, I've got a script that extracts a gif & the audio clip, that way the music & the visual asset are synced.

**Reply by Unthrottled** [reply]
I haven't done it for a lot of them because it is a lot of work and I am a lazy 💩

**Reply by Dragon-0609** [reply]
https://youtu.be/Q5KSkw1KgvQ?t=74

from 1:14 till 1:19

**Comment by Dragon-0609** [comment]
Anime is `Spy x Family`, she is `Yor Forger`. Sleeping (Idle):
![idle](https://yt3.ggpht.com/mWGJAHjRYG9J4tT4iUgEDj0xTRtPgxMhMj_ZH8dNI9xssyyTLHQgTL4DFn9rrCY8Aw4mhRN6paqDGWY=s640-nd-v1-rwa)

If you need sound, you can get it from the episode 5, time is 20:38 -> 20:41

**Reply by Unthrottled** [reply]
I chose one that was more smooth. This one is a bit jittery 😄

**Comment by Dragon-0609** [comment]
The anime is `Science has fallen in love, so we tried to prove it`. She is `Kotonoha Kanade`. Surprised:

[tenor
![surprised](https://c.tenor.com/yzy3aL-UwC0AAAAC/rikei-ga-koi-ni-ochita-no-de-shoumei-shitemita-embarrased.gif)
](https://tenor.com/view/rikei-ga-koi-ni-ochita-no-de-shoumei-shitemita-embarrased-anime-girl-anime-girl-gif-16547021)

---
He is `Professor Ikeda`. Surprised:

[tenor
![surprised 2](https://c.tenor.com/muPOY-A33_YAAAAC/rikei-ga-koi-ni-ochita-no-de-shoumei-shitemita-smile.gif)
](https://tenor.com/view/rikei-ga-koi-ni-ochita-no-de-shoumei-shitemita-smile-mad-anime-gif-16140652)

---
Unwilling:

[tenor
![unwilling](https://c.tenor.com/8QkxIvKjNj0AAAAC/anime-girl.gif)
](https://tenor.com/view/anime-girl-bored-manga-buldeau-gif-20556329)

---
She is `Ayame Himuro`. Blush: 
[tenor
![blush](https://c.tenor.com/rI5Jgr2lQesAAAAC/rikekoi-anime-blush.gif)
](https://tenor.com/view/rikekoi-anime-blush-ayame-himuro-gif-25803932)

NOTE: If somehow you can't see images, select link with dev tools (Chrome). There will be an image inside `<a>`. Select it, after disable `display: none`. When I tried to see images on adding this comment, they weren't shown. So, you can see them by disabling `display: none`.

**Reply by Unthrottled** [reply]
Uploaded! Is this anime any good? I've got it on my watch list, but am waiting until it is done to start.

**Reply by Dragon-0609** [reply]
Yes. I've watched it. At the moment there's a 2nd season in ongoing.

**Comment by Dragon-0609** [comment]
The anime is `To Your Eternity`. He is `Fushi`. Sad:


[link
![sad](https://c.tenor.com/9ieWVqUfmh4AAAAC/to-your-eternity-fushi.gif)
](https://tenor.com/view/to-your-eternity-fushi-sad-sad-face-pout-gif-21518050)

---
idle:
[link
![idle](https://c.tenor.com/BtE2fKJaUUEAAAAC/fushi-fushi-hair-wave.gif)
](https://tenor.com/view/fushi-fushi-hair-wave-white-hair-stare-anime-stare-gif-22032788)

---
The small girl is `March`, the woman is `Hayase`. Angry:

[link
![angry](https://c.tenor.com/hPCKoeXCmLAAAAAC/march-hayase.gif)
](https://tenor.com/view/march-hayase-to-your-eternity-fuumetsu-no-anata-e-fumetsu-no-anata-e-gif-21228919)

---
The old man is `Booze Man`. She is `Pioran`. Laughing
[link
![laughing](https://c.tenor.com/QKbe-PoBipYAAAAd/fushi-and-the-two-old-couple-laugh-fushi-laugh.gif)
](https://tenor.com/view/fushi-and-the-two-old-couple-laugh-fushi-laugh-fushi-anime-anime-laugh-gif-21821192)

---
She is `Rean`. Happy:
[link
![happy](https://c.tenor.com/3r0QmtWPN6UAAAAC/rean-cropp-happy.gif)
](https://tenor.com/view/rean-cropp-happy-cute-fumetsu-no-anata-to-your-eternity-gif-21741736)

**Reply by Unthrottled** [reply]
Uploaded, thanks!

**Reply by Dragon-0609** [reply]
You are welcome

**Comment by Dragon-0609** [comment]
The anime is `A Silent Voice`. He is `Shōya Ishida`. Walking (can be used as idle):

[link
![walking](https://c.tenor.com/cpkRSSElsQAAAAAC/silent-voice-walk.gif)
](https://tenor.com/view/silent-voice-walk-anime-walk-anime-chill-walk-anime-movie-gif-20712245)

---
She is `Shōko Nishimiya`. Waiting (idle):

[link
![waiting](https://c.tenor.com/1m37h3uXPBMAAAAC/a-silent-voice-alone.gif)
](https://tenor.com/view/a-silent-voice-alone-stare-anime-gif-9340232)

---
Preparing cake (can be used as idle):

[link
![preparing](https://c.tenor.com/-CTzlumIy-sAAAAd/anime-bolo.gif)
](https://tenor.com/view/anime-bolo-anivers%C3%A1rio-receita-mecher-gif-18194333)

---
Smile:

[link
![smile](https://c.tenor.com/OH1Hb1nNRRsAAAAd/close-door-smile.gif)
](https://tenor.com/view/close-door-smile-a-silent-voice-shoya-ishida-shouko-nishimiya-gif-17878936)

---
Sad:

[link
![sad](https://c.tenor.com/-LfTFms0v4YAAAAC/koe-no.gif)
](https://tenor.com/view/koe-no-katachi-gif-18428555)

**Comment by Dragon-0609** [comment]
The anime is `Blue Exorcist`. He is `Rin Okumura`. Shocked:

[link
![shocked](https://c.tenor.com/zo8epZGT0FUAAAAC/anime-pointing.gif)
](https://tenor.com/view/anime-pointing-shocked-kingof-gehennaat-mg-ronalds-angry-gif-12647382)

---
Shocked 2:

[link
![shocked](https://c.tenor.com/egCo-A_jbGIAAAAC/blue-exorcist-shocked.gif)
](https://tenor.com/view/blue-exorcist-shocked-gif-25025804)

---
Excited:

[link
![](https://c.tenor.com/ocT-hMAKRWIAAAAC/blue-exorcist.gif)
](https://tenor.com/view/blue-exorcist-dazed-excited-i-cant-believe-it-gif-7475373)

---
The cat is `Kuro`. Excited:

[link
![excited](https://c.tenor.com/bQoszI9YG74AAAAC/excited-kuro.gif)
](https://tenor.com/view/excited-kuro-blue-exorcist-gif-14078900)

---
Perfect:

[link
![perfect](https://c.tenor.com/79E8kJ2-fg8AAAAC/blue-exorcist.gif)
](https://tenor.com/view/blue-exorcist-mindblown-pretty-fabulous-gif-5166807)

---
Idle:

[link
![idle](https://c.tenor.com/XcK1vJKL9CIAAAAC/cat-blue.gif)
](https://tenor.com/view/cat-blue-exorcist-anime-gif-5116153)

---
Crying:

[link
![crying](https://c.tenor.com/WnbQroxCO18AAAAC/anime-cry.gif)
](https://tenor.com/view/anime-cry-crying-sad-gif-9032731)

**Reply by Unthrottled** [reply]
I'm going to implement custom assets, so you can add your own 😃 

You can follow #151 for status updates.

**Comment by Dragon-0609** [comment]
The anime is `Didn't I Say to Make My Abilities Average`, the girl is `Adele von Ascham`.

[link
![source](https://amii-assets.s3.amazonaws.com/visuals/shocked/shockedReactionTwo.gif)
](https://amii-assets.unthrottled.io/assets/view/1478c1faabe80bac7bcd6d0fd9dfa09f)

![picture](https://s1.zerochan.net/Adele.von.Ascham.600.2815026.jpg)

---

The anime is `Shirobako`, the girl is `Aoi Miyamori`.

[link
![source](https://amii-assets.s3.amazonaws.com/visuals/frustrated/frustrated_one.gif)
](https://amii-assets.unthrottled.io/assets/view/380b96c755c9d0855a784c8f51e1515f)

![picture](https://img1.ak.crunchyroll.com/i/spire2/1554dfaa14b433d948290165dd70e7611454618079_full.jpg)

I haven't watched this anime. But if you want to be sure that the source is Shirobako, then go to this [link](http://suresuta.jp/archives/61971). There're screenshots from anime.

---

The anime is `Golden Boy`. I haven't watched this anime, but I'm sure that its name is `Golden Boy`.

[link
![source](https://amii-assets.s3.amazonaws.com/visuals/bored/bored_eight.gif)
](https://amii-assets.unthrottled.io/assets/view/08053f69eb1664dfe45d3b08257543b4)

She doesn't have a name. I searched her name, but couldn't find it in Fandom. In MyAnimeList she's named [Employee E](https://myanimelist.net/character/130149/Employee_E). Maybe somebody who has watched knows her name.

Here are pictures to prove that `Golden boy` is the source. The links below will forward you to the site, which has screenshots of animes. The meme was taken from 1'st episode.

[link to screenshot
![picture](https://ancdn.fancaps.net/16929764.jpg)
](https://fancaps.net/anime/picture.php?/16929764)

[that girl
![picture](https://ancdn.fancaps.net/16929779.jpg)
](https://fancaps.net/anime/picture.php?/16929779)

[that girl (2nd screenshot)
![picture](https://ancdn.fancaps.net/16929816.jpg)
](https://fancaps.net/anime/picture.php?/16929816)

---

The anime is `Yuyushiki`, the girl is `Yuzuko Nonohara`.

[link
![source](https://amii-assets.s3.amazonaws.com/visuals/excited/finna_explode.gif)
](https://amii-assets.unthrottled.io/assets/view/5b389198ad93960c9e35bd5fd0d4a0fa)

![picture](https://i.ytimg.com/vi/p0u5Iwymcsg/hqdefault.jpg)

---

The original game is `Ten Desires`, the girl is `Kyouko Kasodani `.

The gif is from doujinshi (fanmade) `Myourenji Enikki Matome`.

[link
![source](https://amii-assets.s3.amazonaws.com/visuals/frustration/frustration_shitpost.gif)
](https://amii-assets.unthrottled.io/assets/view/a2e71ce3b77c5a14cbec7252a4f65e77)

![picture](https://en13.taadd.com/comics/pic2/47/28655/740955/8f54dece8154d7bd3e0cb0b2b1854b7d.webp?acc=TC3frl8qIlFjfEp6RTKZHQ&exp=1657801261)

[Here](https://knowyourmeme.com/memes/angry-dog-noises) you can read about the meme's origin. If you want to read the doujinshi (fanmade), [here's the link](https://www.gardenmanage.com/statuses/8647349.html).

---

The anime is `Myriad Colors Phantom World`, and the girl is `Mai Kawakami`.

[link
![source](https://amii-assets.s3.amazonaws.com/visuals/pouting/tsudere_pout.gif)
](https://amii-assets.unthrottled.io/assets/view/c9c9ff2eed3dff5c3b9f7c0c033704da)

![picture](https://catf.me/photos/3e65841483d520077ddf5f5e98fd4527.jpg)

**Reply by Unthrottled** [reply]
Updated, thanks!

**Reply by Dragon-0609** [reply]
You're welcome.

**Comment by Dragon-0609** [comment]
Comment above has sources of unknown memes

**Reply by Unthrottled** [reply]
Hey! Thanks for getting this stuff taken care of for me. It's on my radar, I've just got some things I want to finish up first.

**Reply by Dragon-0609** [reply]
Glad to help.

**Comment by Dragon-0609** [comment]
The anime is `Golden boy`. He is `Kintaro Oe`. Smile (for successful build).

![smile](https://user-images.githubusercontent.com/80271099/179372597-b72f341e-32dd-46c1-a35b-d37ef7e04a1a.gif)
